### PR TITLE
XRDDEV-2189 Management requests cs api

### PR DESCRIPTION
**Pull request by ricardas-buc**
https://nordic-institute.atlassian.net/browse/XRDDEV-2189

[XRDDEV-2189]: https://nordic-institute.atlassian.net/browse/XRDDEV-2189?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/common-domain/build.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'java-library'
+    id 'org.springframework.boot'
```
Comment (justasnortal) [inline src/common-domain/build.gradle:3]: Is this plugin needed?

Comment (ricardas-buc) [reply src/common-domain/build.gradle:3]: yes, it injects spring boot dependencies.

File: src/common-management-request/src/main/java/org/niis/xroad/common/managementrequest/verify/decode/util/ManagementRequestVerificationUtils.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+import java.security.cert.X509Certificate;
+
+import static ee.ria.xroad.common.ErrorCodes.X_CERT_VALIDATION;
+import static ee.ria.xroad.common.ErrorCodes.X_INVALID_CLIENT_IDENTIFIER;
+import static ee.ria.xroad.common.ErrorCodes.X_INVALID_REQUEST;
+import static ee.ria.xroad.common.ErrorCodes.translateException;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class ManagementRequestVerificationUtils {
+    private static final CharMatcher MATCHER = CharMatcher.javaIsoControl()
```
Comment (justasnortal) [inline src/common-management-request/src/main/java/org/niis/xroad/common/managementrequest/verify/decode/util/ManagementRequestVerificationUtils.java:56]: More specific name would be easier to read

Comment (ricardas-buc) [reply src/common-management-request/src/main/java/org/niis/xroad/common/managementrequest/verify/decode/util/ManagementRequestVerificationUtils.java]: renamed.

**Review by justasnortal — APPROVED** [review]

**Pull request merged**